packages/drivers/driver-base/src/documentDeltaConnection.ts

```diff
@@ -160,6 +162,24 @@ export class DocumentDeltaConnection
 					this.emit(event, ...args);
 				});
 			}
+
+			// log latency every minute
+			this.trackLatencyTimer = setInterval(() => {
+				const start = Date.now();
+
+				// emit pong event every minute. If latency is longer than 1 min, log separately
+				this.socket.emit("pong", () => {
```

Comment:
Maybe i'm nitpicking, but just to confirm, we don't need to check if this is disposed, right?

Reply:
we already check at the beginning of this function

Reply:
Yes, but the callback code will execute later on (every 60 seconds). So this will be scheduled if it's not disposed. I checked though and this should be fine. In case of disconnects/errors/dispose, this will be unscheduled as the `disconnect` method is wired as appropriate.